Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -263,6 +269,15 @@ - (instancetype)_commonInitializationWithProcessPool:(WebKit::WebProcessPool&)pr
     [self _installVisibilityPropagationViews];
 #endif
 
+#if HAVE(SPATIAL_TRACKING_LABEL)
+    _spatialTrackingView = adoptNS([[UIView alloc] init]);
+    [_spatialTrackingView layer].separatedState = kCALayerSeparatedStateTracked;
+    _spatialTrackingLabel = makeString("WKContentView Label: "_s, createVersion4UUIDString());
+    [[_spatialTrackingView layer] setValue:(NSString *)_spatialTrackingLabel forKeyPath:@"separatedOptions.STSLabel"];
+    [self addSubview:_spatialTrackingView.get()];
+    _page->setSpatialTrackingLabel(_spatialTrackingLabel);
```

Comment:
Will this result in audio emitting from the top-leading corner of the content view?

Reply:
Good question! I'll make sure this gets the equivalent of `width:100%; height:100%`.

Reply:
Or zero-sized but centered.